lib/ferrum/browser/process.rb

```diff
@@ -115,13 +115,15 @@ def restart
 
       private
 
+      attr_accessor :user_data_dir
```

Comment:
We should just put `@user_data_dir = nil` to initializer, because there's no sense in attr_accessor

Reply:
okay, thanks